lib/msal-core/src/Authority.ts

```diff
@@ -150,7 +150,8 @@ export abstract class Authority {
    * Discover endpoints via openid-configuration
    * If successful, caches the endpoint for later use in OIDC
    */
-  public ResolveEndpointsAsync(): Promise<Authority> {
+  public resolveEndpointsAsync(): Promise<Authority> {
+    console.log("resolveEndpointsAsync");
```

Comment:
remove this

Reply:
+1

Reply:
I am assuming you meant the log statement Daryl. One of us will remove this in a future commit.

Reply:
why futures?

Reply:
This is already removed